app/src/main/java/com/benavalli/arcmovies/features/home/presenter/HomeActivityPresenter.kt

```diff
@@ -0,0 +1,44 @@
+package com.benavalli.arcmovies.features.home.presenter
+
+import android.content.Context
+import android.util.Log
+import com.benavalli.arcmovies.api.repository.HomeRepository
+import com.benavalli.arcmovies.base.BasePresenter
+import com.benavalli.arcmovies.domain.UpcomingMovies
+import com.benavalli.arcmovies.features.home.contract.HomeActivityContract
+import javax.inject.Inject
+
+class HomeActivityPresenter
+@Inject
+constructor(private val context: Context, private val repository: HomeRepository) : BasePresenter<HomeActivityContract.View>(),
```

Comment:
is this injected context been used?

Reply:
No, I put it initially because i thought that i would use it,  but i didin't and forgot to remove.

Reply:
removed